client/electron/src/setup.ts

```diff
@@ -213,6 +214,7 @@ export class ElectronCapacitorApp {
   }
 
   updateMountpoint(path: string): void {
+    fs.mkdirSync(path, { recursive: true });
```

Comment:
Something is off here:

When searching for a suitable mounpoint for Windows, we need that the mountpoint path to not exist: https://github.com/Scille/parsec-cloud/blob/4d8960905d968a361163dc763a0cc71bd1be152a/libparsec/crates/platform_mountpoint/src/windows/mount.rs#L162-L175

Here we require that it exist, I've the felling we're going to have concurrency issues

Reply:
But in this case you're mounting the workspace. I'm just creating the folder where the mountpoints are located (`~/Parsec` on Linux). That folder is always the same, it's not deleted when the app closes, and I just make sure it's created right at the start of the app, before we try to mount anything.